build-api/test/sample_data.json

```diff
@@ -0,0 +1,9 @@
+{
+  "deploy_key": "03d77f0d623f0491d6ff5327bb992cbfc9441a0b",
```

Comment:
It's already a dummy deploy key. Github detected this commit and expired this key.